src/datasource.ts

```diff
@@ -287,30 +310,28 @@ export default class IrondbDatasource {
     headers['Accept'] = 'application/json';
     if (irondbOptions['std']['names'].length) {
       const paneltype = irondbOptions['std']['names'][0]['leaf_data']['paneltype'] || 'Graph';
-      options = {};
-      options.url = this.url;
-      if ('hosted' === this.irondbType) {
-        options.url = options.url + '/irondb';
-        options.url = options.url + '/fetch';
-      }
-      options.method = 'POST';
-      if ('standalone' === this.irondbType) {
-        options.url = options.url + '/fetch';
-      }
-      const metricLabels = [];
-      const interval = irondbOptions['std']['interval'];
-      let start = irondbOptions['std']['start'];
-      let end = irondbOptions['std']['end'];
-      start = Math.floor(start - (start % interval));
-      end = Math.floor(end - (end % interval));
-      const reduce = paneltype === 'Heatmap' ? 'merge' : 'pass';
-      const streams = [];
-      const data = { streams: streams };
-      data['period'] = interval;
-      data['start'] = start;
-      data['count'] = Math.round((end - start) / interval);
-      data['reduce'] = [{ label: '', method: reduce }];
       for (let i = 0; i < irondbOptions['std']['names'].length; i++) {
+        options = {};
+        options.url = this.url;
+        if ('hosted' === this.irondbType) {
+          options.url = options.url + '/irondb';
+          options.url = options.url + '/fetch';
```

Comment:
This works fine, but should really be a single operation, with an else into the standalone version to bring those closer together.  nit.